lib/Options/Applicative/MnemonicLanguage.hs

```diff
@@ -0,0 +1,78 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE LambdaCase #-}
+
+{-# OPTIONS_HADDOCK hide #-}
+
+-- |
+-- Copyright: 2020 Input Output (Hong Kong) Ltd., 2021-2022 Input Output Global Inc. (IOG), 2023-2025 Intersect
```

Comment:
I don't think you need to add any copyright notice date prior to current year.

Reply:
good point -> changed